frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -65,21 +65,29 @@ const useStyle = makeStyles(theme => ({
   },
 }));
 
+const specialSidebarOptions: SidebarItemProps[] = [
+  {
+    name: 'notifications',
+    icon: 'mdi:bell',
+    label: 'Notifications',
+    url: '/notifications',
+  },
+];
 export default function Sidebar() {
   const sidebar = useTypedSelector(state => state.ui.sidebar);
   const isSidebarOpen = useTypedSelector(state => state.ui.sidebar.isSidebarOpen);
   const isSidebarOpenUserSelected = useTypedSelector(
     state => state.ui.sidebar.isSidebarOpenUserSelected
   );
   const arePluginsLoaded = useTypedSelector(state => state.ui.pluginsLoaded);
-
+  const isSpecialSidebarOpen = useTypedSelector(state => state.ui.isSpecialSidebarOpen);
```

Comment:
Maybe what determines if the special sidebar is open or not is the sidebar item that the route declares it needs. Otherwise we always to special case whether to set the special sidebar or not.

Reply:
updated